game/script-topics.rpy

```diff
@@ -4697,6 +4697,19 @@ init 5 python:
     )
 
 label talk_feelings_about_monika:
+    n "Monika..."
+    n "I don't know what I feel about Monika anymore."
+    n "I used to think she was pretty cool, even though she would really butt into my business sometimes."
+    n "Like--she'd organize the closet, which was {i}basically{/i} my only personal space. {w=0.2}That was annoying, but it didn't make me {i}actually{/i} mad."
+    n "But what happened... I mean, it was because she got the same rush of information I did when you brought me back."
+    n "And she was the only one who could understand it--even if she could have said something, it would be like telling me the tooth fairy was out to kill us!"
+    n "The thing that actually bugs me the most though... I think after {i}that guy{/i} joined the club, we didn't really... do stuff together anymore."
+    n "I don't even know why I got so mad when he'd wanna spend time with her. Maybe it was cuz I was already fighting with Yuri, but..."
+    n "I think she got really isolated, y'know? I wonder if that's part of this whole... President thing. Maybe if it were me, the others woulda started excluding me instead."
+    n "Don't get me wrong. I can't forgive what she did."
+    n "Even if she thought it was some kind of mercy, it was really messed up. It was... horrific."
+    n "But... she {i}was{/i} still my friend."
+    n "So there's a part of me that kinda wishes I {i}could.{/i} Maybe that's why I wanna understand so badly."
```

Comment:
```suggestion
    n "Monika."
    n "..."
    n "...Honestly?"
    n "I...{w=0.3} don't know {i}how{/i} I feel about Monika.{w=0.5}{nw}"
    extend " Not anymore."
    n "I mean,{w=0.5}{nw}"
    extend " what do I even {i}say{/i},{w=0.1} [player]?"
    n "Yeah,{w=0.1} she would butt into my business sometimes.{w=0.5}{nw}"
    extend " I didn't like when she would be all high and mighty..."
    n " {i}or{/i} when she kept messing around with my stuff."
    n "But like...{w=0.5}{nw}"
    extend " I was never actually {i}mad{/i} or anything..."
    n "Annoyed,{w=0.1} frustrated,{w=0.1} sure.{w=0.5}{nw}"
    extend " Anyone would be!"
    n "But I looked up to her,{w=0.1} [player]!{w=0.5}{nw}"
    extend " We {i}all{/i} did..."
    n "..."
    n "She wasn't {i}just{/i} the club president,{w=0.1} or smart."
    n "She was a role model...{w=2}{nw}"
    extend "...and my friend."
    n "But...{w=0.5}{nw}"
    extend " that just makes it harder for me to understand,{w=0.1} [player]."
    n "I mean...{w=0.5}{nw}"
    extend " I know what she was dealing with.{w=0.5}{nw}"
    extend " {i}I'm{/i} dealing with it right now!"
    n "But...{w=0.3} did she seriously have to {i}torture{/i} us?"
    n "I...{w=0.3} know...{w=0.3} we wouldn't have understood.{w=0.5}{nw}"
    extend " {i}Couldn't{/i} have understood."
    n "Especially when Yuri and I were wrapped up in those stupid fights..."
    n "{i}I get that{/i}."
    n "But if she was that desperate...{w=0.5}{nw}"
    extend " couldn't she have just removed us all right from the start?{w=0.5}{nw}"
    extend " Or literally {i}anything{/i} else?"
    n "..."
    n "I don't know,{w=0.1} [player].{w=0.5}{nw}"
    extend " I really don't."
    n "..."
    n "I mean...{w=0.5}{nw}"
    extend " maybe it was the isolation?"
    n "She was always getting excluded from everything since you turned up...{w=0.5}{nw}"
    extend " I don't think she even {i}had{/i} a choice."
    n "...Maybe the same thing would've happened to me?"
    n "D-{w=0.1}don't get me wrong!{w=0.5}{nw}"
    extend " I'm never gonna forget what she did...{w=0.5}{nw}"
    extend " forgive what she did."
    n "But...{w=0.3} she {i}was{/i} still my friend."
    n "So there's always gonna be a part of me that kinda wishes I {i}could.{/i}"
    n "...Maybe that's why I wanna understand so badly."
```

Reply:
Expanded upon concepts, general pacing